.github/workflows/assign.yaml

```diff
@@ -9,7 +9,7 @@
 name: Assign
 on:
   pull_request_target:
-    types: [assigned, opened, reopened, ready_for_review]
```

Comment:
Please check if you open a PR as draft, then you mark it as ready to review, does it emit `ready_for_review` and `opened`? Otherwise this fix won't work right?

Reply:
Taken out of draft is the `ready_for_review` event only, not opened.  Updated the PR.